recipes/boost/all/conanfile.py

```diff
@@ -16,7 +16,7 @@
 except ImportError:
     from io import StringIO
 
-required_conan_version = ">=1.28.0"
+required_conan_version = ">=1.33.0"
```

Comment:
```suggestion
required_conan_version = ">=1.32.0"
```

even lower ;)

Reply:
CCI policy is to require latest conan version, so 1.33 is okay, no need to change

Reply:
Thanks, I prefer to avoid triggering a rebuild for this trivial change 😄

Reply:
It's not very important. But I disagree, `required_conan_version` is a useful information. It avoids cryptic errors for consumers. Moreover, think to beginners, conan is already complex, so let be honest, few people read the all documentation.

Reply:
It's requiring conan 1.33.0 now. Isn't that fine?

Reply:
But sure, if this is the only modification, let's skip it since it takes 4 of 5 hours to build (which is not too long actually for so many versions).

Reply:
The biggest problem is that the build infrastructure is a bit unstable when building boost. I needed to trigger a few rebuilds before it could complete.